packages/vm/lib/evm/interpreter.ts

```diff
@@ -191,27 +187,6 @@ export default class Interpreter {
       codeAddress: this._eei._env.codeAddress,
     }
 
-    // Create opTrace for debug functionality
-    let hexStack = []
-    hexStack = eventObj.stack.map((item: any) => {
-      return '0x' + new BN(item).toString(16, 0)
-    })
-
-    const name = eventObj.opcode.name
```

Comment:
Is it possible to regenerate this data in the `step` event? We have access to this `eventObj` in the event right? Then it is OK if all this is removed in this context.

Reply:
Yes, you can generate the same data.

Reply:
This change here changes the structure of the step event as mentioned in #1199

Reply:
Where's the change? I really don't get it.

Reply:
If one would listen to `step` like this;

```
vm.on('step', (obj: any) => {
// do stuff
let stack = obj.hexStack
// do stuff with stack (is now undefined)
})
```

Then this code now does not work anymore, since `hexStack` is not available anymore (and this `opTrace` thing is also not available anymore).

Reply:
That object is not passed to the step handler. Take a look at line 232:

```ts
return this._vm._emit('step', eventObj)
```

Only `eventObj` is passed, which I didn't modify.